.github/workflows/test_and_build.yml

```diff
@@ -48,7 +48,54 @@ jobs:
       - run: zig/zig build test
       - run: ./scripts/install.sh
 
+  fuzz_lsm_manifest_log:
+    name: 'Fuzz LSM ManifestLog'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./scripts/install_zig.sh
+      - run: zig/zig build fuzz_lsm_manifest_log
```

Comment:
How many time is this run in CI? Do we want to run it with a fixed seed?

Reply:
Just once. It is sort of a canary, i.e. make sure that the fuzzer builds and succeeds a test.
The intent isn't to use GitHub as our fuzzer infrastructure, it is just to make sure there are no low-hanging bugs introduced by a PR.

Re: fixed seed — I was thinking that fuzzer errors will (hopefully!) be rare enough that it wouldn't matter, and varying the seed can give us a tiny bit extra coverage, but I'm open to changing it.

Reply:
I guess let's leave it random until it's a problem.

Reply:
Thinking about this more I think you're right, we should use a fixed seed so that we never have the PR pass CI, then fail it only after it is merged.